charts/redpanda/Chart.yaml

```diff
@@ -23,7 +23,7 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 5.6.55
+version: 5.6.56
```

Comment:
nit: helm-docs should be executed so that the README reflects the version changes

Reply:
Ok I will do that :)